src/main/java/org/truffleruby/language/arguments/RubyArguments.java

```diff
@@ -80,11 +94,38 @@
         packed[ArgumentIndicies.SELF.ordinal()] = self;
         packed[ArgumentIndicies.BLOCK.ordinal()] = block;
 
+        // TODO: We need to somehow set it to "flattened" or something else
+        //   if it's flattened.
+        packed[ArgumentIndicies.KW_ARGS_OPTIMIZABLE_FLAG.ordinal()] = null; // Set to null for now
```

Comment:
Maybe create an object to represent a unique keyword argument signature, and pass that as a marker?

Reply:
Something like, `Map<String, Integer>` ?